src/android/jsc_override.cpp

```diff
@@ -143,6 +145,7 @@ static JSGlobalContextRef create_context(JSContextGroupRef group, JSClassRef glo
 
 static void swap_function()
 {
+    __android_log_print(ANDROID_LOG_INFO, "svap_function", "BEGIN");
```

Comment:
Maybe just log as debug? Same for others

Reply:
I should have been explicit about it: I plan to remove all logging before releasing.